astropy/coordinates/tests/test_transformations.py

```diff
@@ -689,3 +689,13 @@ def test_combine_affine_params(first, second, check):
         assert result[1] is None
     else:
         assert_allclose(result[1].xyz, check[1].xyz)
+
+
+def test_cds_with_coordinates_transformations():
+    """See https://github.com/astropy/astropy/issues/14580."""
+    from astropy.coordinates import SkyCoord
```

Comment:
The fact that `SkyCoord` is not being imported in this file suggests that perhaps `astropy/coordinates/tests/test_celestial_transformations.py` would be a more suitable location.